pkg/providers/anthropic/chat_stream.go

```diff
@@ -2,15 +2,102 @@ package anthropic
 
 import (
 	"context"
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
 
 	"glide/pkg/api/schemas"
 	"glide/pkg/providers/clients"
 )
 
+type Client struct {
```

Comment:
In Golang uniquely, compared to other programming languages, allows struct/object methods can be separate across go files in the module. For example, in case of Anthropic client is spread across these three files:
 
<img width="297" alt="Screenshot 2024-03-19 at 22 25 12" src="https://github.com/EinStack/glide/assets/9402690/b64f9f07-aa49-4699-8fe6-6fb343e3d408">

Specifically, the client is defined in https://github.com/EinStack/glide/blob/develop/pkg/providers/anthropic/client.go#L22

and it has already pulling a lot of useful values like [API key (e.g. c.config.APIKey)](https://github.com/EinStack/glide/blob/develop/pkg/providers/anthropic/chat.go#L102C51-L102C66) and [a bunch of other configurations](https://github.com/EinStack/glide/blob/develop/pkg/providers/anthropic/chat.go#L24-L49). So these three files are really worth checking.